packages/Webkul/Core/src/Core.php

```diff
@@ -605,21 +605,81 @@ public function formatPrice($price, $currencyCode = null)
 
         $formatter->setAttribute(\NumberFormatter::FRACTION_DIGITS, $currency->decimal ?? 2);
 
-        if (! $currency) {
+        if (!$currency) {
             return $formatter->formatCurrency($price, $currencyCode);
         }
 
         if ($symbol = $currency->symbol) {
             if ($this->currencySymbol($currency) == $symbol) {
-                return $formatter->formatCurrency($price, $currency->code);
-            }
 
+                return $this->formatPriceWithSymbol($price, $currency, $currency->symbol, $formatter);
+                // _________________________________________
+
+                // return $formatter->formatCurrency($price, $currency->code);
+            }
             $formatter->setSymbol(\NumberFormatter::CURRENCY_SYMBOL, $symbol);
 
-            return $formatter->format($this->convertPrice($price));
+            return $this->formatPriceWithSymbol($price, $currency, $currency->symbol, $formatter);
+            // _______________________
+            // $formatter->format($this->convertPrice($price));
+
+        }
+        return $this->formatPriceWithSymbol($price, $currency, $currency->code, $formatter);
+        // _______________________________________
+        // return $formatter->formatCurrency($price, $currency->code);
+    }
+
+    /**
+     *
+     * customize the thousand separator and decimal separator.
+     * Additionally, repositioning of the currency symbol and the inclusion of an HTML tag for further styling.
+     * @param  float  $price
+     * @param Webkul\Core\Models\Currency $currency
+     * @param string $symbol
+     * @param \NumberFormatter $formatter 
```

Comment:
https://laravel.com/docs/10.x/contributions#phpdoc

Reply:
Thanks for sharing the link! We have followed it.